src/app/@shared/card-vendor/card-vendor.component.html

```diff
@@ -1,18 +1,18 @@
-<nb-card size="small" class="vendor-card clickable-card" [routerLink]="globalRoutes.VENDOR_PAGE_ROOT + '/' + item.objectID">
-    <nb-card-header
-        ><h5 class="text-heading-1">{{ item.title }}</h5></nb-card-header
-    >
-    <nb-card-body class="card-body">
-        <img
-            class="card-image"
-            [src]="
-                item.image ? item.image : 'https://upload.wikimedia.org/wikipedia/commons/thumb/a/ac/No_image_available.svg/1200px-No_image_available.svg.png'
-            "
-            alt="name"
-        />
-    </nb-card-body>
-    <nb-card-footer>
-        <h6 class="desc">{{ item.description }}</h6>
-        <p>{{ item.address }}, {{ item.city }}, {{ item.country }}</p>
-    </nb-card-footer>
-</nb-card>
+<section class="card vendor-card clickable-card" [routerLink]="globalRoutes.VENDOR_PAGE_ROOT + '/' + item.objectID">
+    <header>
```

Comment:
The HTML `<header>` element represents introductory content, typically a group of introductory or navigational aids. It may contain some heading elements but also a logo, a search form, an author name, and other elements.

Reply:
Perhaps have vendors as <article> tags with <header> in them? It includes logo, vendor title, location and description, I think <header> semantically makes sense in this context.